### PR TITLE
Setup Webpack, Babel, PostCSS, ESLint and run scripts

### PR DESCRIPTION
**Pull request by YangStephen**
Introduces Webpack, Babel, PostCSS, ESLint and modifies our run scripts to be dependent on Webpack.

A few notes:
Webpack helps package up our application rather than relying on the default npm build. We also modify our build scripts to run the built version so instead of relying on our local code, we are always building / running what will be a production build in the future.

PostCSS is a CSS processor that allows us to extend functionality. Right now, theres a postcss config file while imports a group of plugins. From the plugins, we can import css classes from other css files, and we can also use a inherit function:
```
.Blue{
    color: blue;
}

.RandomCSSClass{
   @inherit: .Blue; << This class now has color: blue in the object. We can override it easily by putting a color: Red; under it.
}
```

### CODE REVIEW

File: .eslineignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/src/index.js
```
Comment (Dendrimer) [inline .eslineignore]: new line?

File: postcss.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const postcssPresetEnv = require('postcss-preset-env');
```
Comment (Dendrimer) [inline postcss.config.js]: why is this the only variable being explicitly defined like this?

**Comment by Dendrimer** [comment]
Should you also be committing the initial yarn.lock file?

File: postcss.config.js (hunk 2 above)
Comment (YangStephen) [reply postcss.config.js]: Resolved

**Review by Dendrimer — APPROVED** [review]
good job :shipit:

**Pull request merged**